### PR TITLE
Add 2020 cfp

### PR DESCRIPTION
**Pull request by dcwalk**

### CODE REVIEW

File: _includes/2020-hero.html [hunk 1]
```diff
@@ -1,8 +1,10 @@
 <div class="splash-hero pt-m pb-m">
   <div class="container container-full-width">
   <header>
-    <h1 class="heading-1 mb-0 mt-0">Our Networks</h1>
-    <h2 class="heading-3">August 2020</h2>
+    <h1 class="heading-1 mb-0 mt-0">Our Networks in Uncertain Times</h1>
+    <h2 class="heading-3">August 7-9 2020</h2>
```
Comment (garrying) [inline _includes/2020-hero.html]:
```suggestion
    <h2 class="heading-3">August 7–9, 2020</h2>
```

File: index.md [hunk 2]
```diff
@@ -6,10 +6,26 @@ redirect_from:
...
+**This is a call to face the uncertainty together—come meet us online.** We’re looking for a range of virtual activations through the month of July leading up to and including the weekend of August 7-9! There will be space for familiar formats, including streamed talks and virtual workshops, but we are especially interested in activations that break free from the bounds of an “online conference.” For example, you could host a durational study group, run a meetup in a game world, exhibit an art project using a platform in some unexpected way, experiment with collective forms of distributed expression (à la Twitch Plays Pokémon). We are excited to draw on the possibilities for shared online experiences that a remote event affords. Leaning into new forms of interaction that build collective experiences at a distance, we can collectively cultivate virtual experiences that feel intimate, exploratory, and expansive.
 
-Please don't hesitate to [get in touch](mailto:{{ site.email }}). 
+## Submission Deadlines
```
Comment (garrying) [inline index.md]:
```suggestion
## Submission deadline
```

File: index.md [hunk 3]
```diff
@@ -6,10 +6,26 @@ redirect_from:
...
+
+The deadline for proposals is **June 1, 2020**. We highly encourage you to email us or attend an open hour if you have questions about formats and activations—we’re here to offer support. Submissions should [follow our template](https://raw.githubusercontent.com/ournetworks/2020-submissions/master/.github/ISSUE_TEMPLATE/submission.md) and be [submitted as a GitHub issue](https://github.com/ournetworks/2020-submissions/issues/new) in our submissions repository. There will be additional opportunities closer to and during the event to put forward ideas and actively participate. We encourage you to sign up to our mailing list to hear more.
+
+<a href="https://github.com/ournetworks/2020-submissions/issues/new" class="button" target="_blank" rel="noopener">Submit your proposal</a>
```
Comment (garrying) [inline index.md]:
```suggestion
<a href="https://github.com/ournetworks/2020-submissions/issues/new?template=submission.md&title=%5BSubmission+Title%5D" class="button" target="_blank" rel="noopener">Submit your proposal</a>
```
`issues/new` opens to a blank issue. I'm not sure if the flow was the same last year.

File: index.md [hunk 4]
```diff
@@ -6,10 +6,26 @@ redirect_from:
...
+
+_First time GitHub users will need to [create a free account](https://github.com/join) before they can [open an issue](https://help.github.com/en/github/managing-your-work-on-github/creating-an-issue)._
+
+## Who can submit?
+
+Anyone interested in how we can build better online futures! Our Networks is excited for sessions that centre the voices of those less often heard at technical events. We encourage BIPOC and LGBTQ2S folks to lead sessions and acknowledge the multiple marginalizations those with intersecting identities face. For 2020 we are also seeking to support those voices in their preferred language if you are interested in leading a session in a language other than English please indicate so in your submission.
```
Comment (garrying) [inline index.md]:
```suggestion
Anyone interested in how we can build better online futures! Our Networks is excited for sessions that centre the voices of those less often heard at technical events. We encourage BIPOC and LGBTQ2S folks to lead sessions and acknowledge the multiple marginalizations those with intersecting identities face. For 2020 we are also seeking to support those voices in their preferred language. If you are interested in leading a session in a language other than English please indicate so in your submission.
```

File: index.md [hunk 5]
```diff
@@ -6,10 +6,26 @@ redirect_from:
...
 
-Be assured, we're plotting a course for Our Networks 2020 to continue. As we proceed we also want to hear your thoughts, concerns, and hopes for how we can strengthen our networks in uncertain times. 
+**This is a call to face the uncertainty together—come meet us online.** We’re looking for a range of virtual activations through the month of July leading up to and including the weekend of August 7-9! There will be space for familiar formats, including streamed talks and virtual workshops, but we are especially interested in activations that break free from the bounds of an “online conference.” For example, you could host a durational study group, run a meetup in a game world, exhibit an art project using a platform in some unexpected way, experiment with collective forms of distributed expression (à la Twitch Plays Pokémon). We are excited to draw on the possibilities for shared online experiences that a remote event affords. Leaning into new forms of interaction that build collective experiences at a distance, we can collectively cultivate virtual experiences that feel intimate, exploratory, and expansive.
```
Comment (garrying) [inline index.md]:
```suggestion
**This is a call to face the uncertainty together—come meet us online.** We’re looking for a range of virtual activations through the month of July leading up to and including the weekend of August 7–9! There will be space for familiar formats, including streamed talks and virtual workshops, but we are especially interested in activations that break free from the bounds of an “online conference.” For example, you could host a durational study group, run a meetup in a game world, exhibit an art project using a platform in some unexpected way, experiment with collective forms of distributed expression (à la Twitch Plays Pokémon). We are excited to draw on the possibilities for shared online experiences that a remote event affords. Leaning into new forms of interaction that build collective experiences at a distance, we can collectively cultivate virtual experiences that feel intimate, exploratory, and expansive.
```
Diff: en dash between 7 and 9.

**Comment by dcwalk** [comment]
@garrying final review pls :pray:

**Review by garrying — APPROVED** [review]

**Pull request merged**